package.json

```diff
@@ -7,8 +7,8 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "postinstall": "npm run bootstrap",
-    "lint": "eslint . --quiet",
-    "lint:fix": "eslint . --quiet --fix",
+    "lint": "eslint . --quiet && prettier --check .",
```

Comment:
So both those things come with their own disadvantages. There’s some detail about them both here: https://prettier.io/docs/en/integrating-with-linters.html#notes

I don’t think theres anything wrong with running them separately. We have the [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) setup to avoid any conflicts between the two.

Reply:
Sounds good, thanks.